pkg/agent/testdata/RawUbuntuContainerd/line70.sh

```diff
@@ -494,6 +494,14 @@ ensureKubelet() {
         logs_to_events "AKS.CSE.ensureKubelet.setKubeletNodeIPFlag" setKubeletNodeIPFlag
     fi
 
+    if semverCompare ${KUBERNETES_VERSION:-"0.0.0"} "1.32.0"; then
+        tee "/etc/systemd/system/kubelet.service.d/10-watchdog.conf" > /dev/null <<'EOF'
+[Service]
+WatchdogSec=60s
+NotifyAccess=all
```

Comment:
I guess we can probably get rid of the NotifyAccess line now that we're back to having systemd fork kubelet directly